### DISCUSSION TITLE
Pnp Modern Search Preselected Filter

### DISCUSSION THREAD

**Discussion by rosamartinamilazzo25** [microsoft-search/pnp-modern-search#3144] [Q&A]
**Version used**
Ex: 4.8

**Describe the bug**
Good morning, I am using the PnP Modern Search web part, specifically the Result and Filter web parts. I am experiencing a bug with the filters, let me explain it better: I have a filter with a checkbox containing two values, and I would like to have one of those two values preselected. I have tried making the filters static and adding a query string ([{"filterName":"RefinableString03","values":[{"name":"APERTA","value":"ǂǂ415045525441","operator":0}],"operator":"and"}]), but the documentation is not very clear on this point, and I am unsure where to insert it. Does anyone know how to solve this issue? Thank you.

**Comment by kasperbolarsen** [comment]
It sound like Filter deep linking is what you are looking for? see https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/#filter-types-static-filter-versus-refiner-filter
